test/performance/driver.F90

```diff
@@ -21,6 +21,7 @@ program performance_test
   !> Runs MICM under prescribed conditions and collect performance metrics
   subroutine run_test( )
 
+    use constants,                     only : kNumberOfGridCells
```

Comment:
do we need the grid point information in the driver code?

Reply:
it's used to set up the array of environmental conditions here:
https://github.com/NCAR/micm/blob/637bcd3d52f17c41a22dce6f2d06e969a8f991e5/test/performance/driver.F90#L41-L43

and the array of species concentrations here:
https://github.com/NCAR/micm/blob/637bcd3d52f17c41a22dce6f2d06e969a8f991e5/test/performance/driver.F90#L64-L66

Reply:
Oops! Apologize that I did not catch those declarations. That makes sense to me now.